modules/social_features/social_event/social_event.install

```diff
@@ -1621,3 +1621,37 @@ function social_event_update_8904() {
 function social_event_update_8905() {
   \Drupal::service('module_installer')->install(['social_event_addtocal']);
 }
+
+/**
+ * Show thumbnail on File fields when attachment is an image.
+ */
+function social_event_update_8906() {
+  $config_file = drupal_get_path('module', 'social_event') . '/config/static/core.entity_view_display.node.event.default_8906.yml';
+
+  if (is_file($config_file)) {
+    $settings = Yaml::parse(file_get_contents($config_file));
+    if (is_array($settings)) {
+      $config = \Drupal::configFactory()
+        ->getEditable('core.entity_view_display.node.event.default');
+
+      $config->setData($settings)->save(TRUE);
+    }
+  }
+}
+
+/**
+ * Add image file extensions to be added as attachments.
+ */
+function social_event_update_8907() {
+  $config_file = drupal_get_path('module', 'social_topic') . '/config/static/field.field.node.event.field_files_8907.yml';
```

Comment:
Hi @sjoerdvandervis @navneet0693 shouldn't this be 'social_event' instead of topic?

Reply:
You are right @ronaldtebrake.